packages/cli/src/index.ts

```diff
@@ -86,13 +86,16 @@ const main = async () => {
     log(`Started JSON RPC Server address=http://${rpcAddr}:${args.rpcPort}`)
   }
 
-  process.on('SIGINT', async () => {
-    console.log('Caught close signal, shutting down...')
-    await portal.stop()
-    if (metricsServer?.listening === true) {
-      metricsServer.close()
-    }
-    process.exit()
+  process.on('uncaughtException', (err) => {
+    console.error('Uncaught Exception:', err)
```

Comment:
This should be it's own listener and not wrap the `sigint` listener.  That's what is causing that memory leak warning you reported.